subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginIntegrationTest.groovy

```diff
@@ -0,0 +1,46 @@
+package org.gradle.profiler.studio.plugin
+
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase
+import org.gradle.profiler.client.protocol.Server
+import org.gradle.profiler.client.protocol.ServerConnection
+import org.gradle.profiler.client.protocol.messages.StudioRequest
+import org.gradle.profiler.client.protocol.messages.StudioSyncRequestCompleted
+
+import java.time.Duration
+
+import static org.gradle.profiler.client.protocol.messages.StudioRequest.StudioRequestType.EXIT
+import static org.gradle.profiler.client.protocol.messages.StudioRequest.StudioRequestType.SYNC
+import static org.gradle.profiler.client.protocol.messages.StudioSyncRequestCompleted.StudioSyncRequestResult.SUCCEEDED
+import static org.gradle.profiler.studio.plugin.client.GradleProfilerClient.PROFILER_PORT_PROPERTY
+
+/**
+ * Note: This class uses Junit3 so every test should start with "test"
```

Comment:
Why JUnit 3? I wonder if `BasePlatformTestCase` also works with JUnit 4, the Javadoc on `UsefulTestCase` seems to claim that. Though I suppose it is not so important.

Reply:
You are right, it works with Junit4. I fixed that in: https://github.com/gradle/gradle-profiler/pull/366